pkg/morph/deploy/deploy.go

```diff
@@ -645,3 +666,25 @@ func encodeFloatConfig(v float64) []byte {
 func encodeBoolConfig(v bool) []byte {
 	return stackitem.NewBool(v).Bytes()
 }
+
+// returns actor.TransactionCheckerModifier which sets current NeoFS epoch as
+// nonce of the transaction and makes it valid 100 blocks after Sidechain block
+// when the epoch began.
+func neoFSRuntimeTransactionModifier(neoFS NeoFS) actor.TransactionCheckerModifier {
+	return func(r *result.Invoke, tx *transaction.Transaction) error {
+		err := actor.DefaultCheckerModifier(r, tx)
+		if err != nil {
+			return err
+		}
+
+		neoFSState, err := neoFS.CurrentState()
+		if err != nil {
+			return fmt.Errorf("get current NeoFS network state: %w", err)
+		}
+
+		tx.Nonce = uint32(neoFSState.CurrentEpoch)
+		tx.ValidUntilBlock = neoFSState.CurrentEpochBlock + 100
```

Comment:
So with a 240-block epoch we can't do anything at all after N+100?

Reply:
seems like that

Reply:
made valid for one epoch